aiohttp/__init__.py

```diff
@@ -176,6 +186,7 @@
     'ThreadedResolver',
     # signals
     'Signal',
+    # streams
```

Comment:
It's unrelated

Reply:
It's related to the imports cleanup, and makes __all__ consistent with imports here.

Reply:
It's unrelated to re-exports.

Reply:
ok